styles/hydrogen.less

```diff
@@ -111,6 +111,12 @@ svg:first-child {
 .multiline-container {
   display: flex;
   border: solid 1px @base-border-color;
+  &[hydrogen-wrapoutput="true"] {
```

Comment:
Maybe better to be `wrapOutput` (note the case of `Output`) even tho the Less might be case-insensitive.

Reply:
If less isn't case sensitive I will change, however if it is I will leave it as is

Reply:
`=` operator is case-insensitive, and I see what you want.